.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,6 +28,26 @@ For new packages please briefly describe the package or provide a link to its ho
   - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
 - [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).
 
+###### Getting started with reviewing this package
+
+<!--
+Put in this section any additional comment that might help potential reviewers to review your PR. Describe for instance:
+- If you expect the PR to be simple to review or not (say if it's just a simple package update with few dependencies, or a package with many dependencies that have high chances to cause breakage). This can help reviewiers to easily identify PR that can be merged quickly.
+- The instructions to check if the bug is fixed etc, which binary should be used to start the program, which command can be run to check basic functionalities... This can help people unexperienced with the software to get started.
+Make sure to update the placeholders in the intructions below (in upper case letter). Note that you might need to first submit the PR and then edit it to change PUT_HERE_THE_PR_NUMBER_CF_URL.
+-->
+
+To start reviewing this PR, you can first run [nixpkgs-review](https://github.com/Mic92/nixpkgs-review):
+```
+$ # first cd to a clone of nixpkgs, or clone it, e.g. in your $HOME: git clone https://github.com/NixOS/nixpkgs
+$ nix run nixpkgs#nixpkgs-review -- pr PUT_HERE_THE_PR_NUMBER_CF_URL
+```
+
+Then, basic functionalities can be tested using:
+```
+$ ./result/YOUR_PROGRAM/bin/YOUR_BINARY
+```
+
```

Comment:
Isn't this part mostly repeating the checkbox further above?

Reply:
BTW, the path would be `./result/bin/YOUR_BINARY`

Reply:
Similarly, `nixpkgs-review` already did have a command how to run it above near the checkbox, so I'm (personally) not sure about the overall structure of the whole page we're getting here.

Reply:
Sorry, just saw your message. Well the commands you mention are for the person submitting the PR, not the reviewer itself. The goal of this PR is to provide instructions to the **reviewer** as well to get started quickly, that should be filled by the person making the PR. Well, `YOUR_PROGRAM` is at least needed if the PR changes multiple programs.

If you review so often packages that you know by heart the `nixpkgs-review` command, together with all the programs in nixpkgs and how to test them… that’s great, but it is not the case of everybody here (as a beginner, I might not be even aware that nixpkgs-review can help me here), am my guess is that people would be more inclined to review if we help them as much as possible in this boring and time consuming task. Like, let’s start with a random PR I can see: https://github.com/NixOS/nixpkgs/pull/257997 Do you know the sink-rotate software? I don't. So if I want to know if I am brave enough to review it, I would need to:
1. google the name of the software
2. struggle to find the website because "sink-rotate" is poorly referenced and appears last in the page of google in the rust crate website…
3. then I realize it is somehow related to rust… is it a program? A library? The description mentions "Command that sets "next" PipeWire audio sink as default"… so it is certainly a command-line interface, but there is also maybe a library that I should test as well as it appears in the website containing rust crates? After reading more the crate page, I can see that it is not a library. Good to know, less things to test.
4. From the website, it seems like I need to have pipewire installed… Hopefully, I know and even have pipewire installed, but it is maybe not the case of everyone… so a new comer would have to figure out all details about setting up pipewire, and the potential implications that it may have on its system (and for instance if you need some hardware/GPU to test something, you want to be aware directly, not after 30mn of research and compilation).
5. The description of the package on crate only says "Command that sets "next" PipeWire audio sink as default", but it is not really clear what it means. Is it just a shortcut to quickly change the audio output? So does it mean that I can't test this package if I only have one audio output ?
6. To answer my question, I go to the source of the package to find more details. I find the github page https://github.com/mightyiam/sink-rotate… unfortunately it does not provide **any** help on how to run this package. 

I don't know what this package is doing, nor how to use it, nor if I have the proper hardware to use it, and I even forgot how to use nix-review to quickly give it a try… so I guess 90% of potential reviewers will have just lost 10mn trying to figure this out before realizing that with a single audio source, they cannot test this package. 10mn lost for no result at all. Now, let’s say that I’m lucky, I have pipewire installed and two audio sources, then the fight continues:
7. I need to know about the existence of nixpkgs-review: beginners would just be like "ok, how do I compile it now"?
8. I need to know the command to run it. Since I forgot them, I need to google nixpkgs review, and find the command line to run (one minute lost)
9. I compile the program, I need to wait… how long? Maybe it requires 30mn of compilation, I don't know before trying.
10. Finally the program is compiled. I need to know the name of the executable to run. If I run ls, I might find 10 binaries… which one to test?
11. I found the binary I want to test. How to run it properly? What should I type? What should I expect if it is working correctly?
12. Maybe the PR is fixing a specific bug. How could I try to reproduce the bug to see if the bug is actually fixed? I don't even know the software, so little change I can know the bugs of it.

Anyway, at that step it require more energy that I can to spend on this… so little change I’ll review this package. With proper instructions that I can copy paste and that takes me literally 10s to type… there is more chance I would try to review. It’s like this article that sayed that if you enable double side printing by default, people will save paper, just because they are too lazy to enable if it is not enabled by default. Reviewers are precious, let’s not waste their time.